ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -224,363 +301,231 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
-      )
-      @Resolution("Contact the participant operator.")
-      object LedgerConfigurationNotFound
-          extends ErrorCode(
-            id = "LEDGER_CONFIGURATION_NOT_FOUND",
-            ErrorCategory.InvalidGivenCurrentSystemStateResourceMissing,
+    @Explanation(
+      "This error is emitted when a submitted ledger API command specifies an invalid deduplication period."
+    )
+    @Resolution(
+      "Inspect the error message, adjust the value of the deduplication period or ask the participant operator to increase the maximum."
+    )
+    object InvalidDeduplicationPeriodField
+        extends ErrorCode(
+          id = "INVALID_DEDUPLICATION_PERIOD",
+          ErrorCategory.InvalidGivenCurrentSystemStateOther,
+        ) {
+      case class Reject(_reason: String, _maxDeduplicationDuration: Duration)(implicit
+          loggingContext: ContextualizedErrorLogger
+      ) extends LoggingTransactionErrorImpl(
+            cause = s"The submitted command had an invalid deduplication period: ${_reason}"
           ) {
-
-        case class Reject()(implicit
-            loggingContext: ContextualizedErrorLogger
-        ) extends LoggingTransactionErrorImpl(
-              cause = "The ledger configuration is not available."
-            )
+        override def context: Map[String, String] =
+          super.context + ("max_deduplication_duration" -> _maxDeduplicationDuration.toString)
       }
     }
 
-    @Explanation("""This error occurs if the Daml transaction fails due to an authorization error.
-        |An authorization means that the Daml transaction computed a different set of required submitters than
-        |you have provided during the submission as `actAs` parties.""")
-    @Resolution("This error type occurs if there is an application error.")
-    object AuthorizationError
+    @Explanation("""The supplied offset could not be converted to a binary offset.""")
+    @Resolution("Ensure the offset is specified as a hexadecimal string.")
+    object NonHexOffset
         extends ErrorCode(
-          id = "DAML_AUTHORIZATION_ERROR",
+          id = "NON_HEXADECIMAL_OFFSET",
           ErrorCategory.InvalidIndependentOfSystemState,
         ) {
-
-      case class Reject(override val cause: String)(implicit
-          loggingContext: ContextualizedErrorLogger
-      ) extends LoggingTransactionErrorImpl(
-            cause = cause
+      case class Error(
+          fieldName: String,
+          offsetValue: String,
+          message: String,
+      )(implicit
+          override val loggingContext: ContextualizedErrorLogger
+      ) extends BaseError.Impl(
+            cause =
+              s"Offset in ${fieldName} not specified in hexadecimal: ${offsetValue}: ${message}"
           )
     }
   }
 
-  @Explanation("""This error occurs if there was an unexpected error within the interpreter""")
+  @Explanation("""This error occurs if there was an unexpected error in the Ledger API.""")
```

Comment:
Nit-picking:
Inconsistent capitalization in `Ledger API` vs. `ledger Api`.

Reply:
You mean `Ledger API Server`?